### PR TITLE
feat/components/toggles

### PR DESCRIPTION
**Pull request by Aniket1026**
Below are the changes I made . Here I added to toggle buttons which I think are useful in app or websites to toggle anything in general . If there is something I can improve on or any mistake I made in general  please do tell me . Thank you 😄😄

https://user-images.githubusercontent.com/92184189/181916605-b7cf7cc4-e3ab-4f44-8e46-9a6b1fe3e81a.mp4

### CODE REVIEW

**Comment by markmead** [comment]
Hey @Aniket1026 thanks for this, but I'm not quite sure you understand what HyperUI is. It's a Tailwind CSS component library and therefore components are written with Tailwind CSS.

File: data/components/application-ui-button-groups.mdx [hunk 1]
```diff
@@ -15,6 +15,11 @@ components:
     title: Action Group
   4:
     title: Action Group with Dropdown
+  5:
```
Comment (markmead) [inline data/components/application-ui-button-groups.mdx:18]: These would be under a new collection called `application-ui-toggles.mdx`.

File: public/build.css [hunk 2]
```diff
@@ -1 +1,4 @@
...nt-weight:700}.lg\:uppercase{text-transform:uppercase}.lg\:tracking-wide{letter-spacing:.025em}.lg\:text-gray-500{--tw-text-opacity:1;color:rgb(107 114 128/var(--tw-text-opacity))}.lg\:opacity-0{opacity:0}.lg\:transition{transition-property:color,background-color,border-color,fill,stroke,opacity,box-shadow,transform,filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter;transition-property:color,background-color,border-color,text-decoration-color,fill,stroke,opacity,box-shadow,transform,filter,backdrop-filter,-webkit-text-decoration-color,-webkit-backdrop-filter;transition-timing-function:cubic-bezier(.4,0,.2,1);transition-duration:.15s}.lg\:transition-all{transition-property:all;transition-timing-function:cubic-bezier(.4,0,.2,1);transition-duration:.15s}.group:hover .lg\:group-hover\:absolute{position:absolute}.group:hover .lg\:group-hover\:relative{position:relative}.group:hover .lg\:group-hover\:opacity-0{opacity:0}.group:hover .lg\:group-hover\:opacity-100{opacity:1}}@media (min-width:1280px){.xl\:max-w-\[1348px\]{max-width:1348px}}
```
Comment (markmead) [inline public/build.css:1]: This file is re-compiled on every deployment and therefore any changes would be removed.

File: public/components/application-ui-button-groups/5.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+  <label class='switch' >
+              <input type='checkbox' />
+              <span class='slider' > </span>
+          </label>
```
Comment (markmead) [inline public/components/application-ui-button-groups/5.html:4]: The formatting isn't correct and you haven't used Tailwind CSS.

File: public/components/application-ui-button-groups/6.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+     <label class='toggle' >
+              <input type='checkbox' />
+              <span class='slide' ></span>
+          </label>
```
Comment (markmead) [inline public/components/application-ui-button-groups/6.html:4]: The formatting isn't correct and you haven't used Tailwind CSS.

**Review by markmead — CHANGES_REQUESTED** [review]

**Comment by Aniket1026** [comment]
Sure I'll make changes using tailwind Css . I just started contributing so I was confuse about how to make changes but thanks for telling me .  I'll now  use tailwind to make the same changes 😄

**Comment by markmead** [comment]
No problem @Aniket1026 I assume that as it's a Tailwind CSS component library it would be obvious, but I'll update the docs.

**Comment by markmead** [comment]
Rember to re-request my review when you have something @Aniket1026, it looks like a lot of progress has been made!

**Comment by Aniket1026** [comment]
Yeahh sure I'll take care of that from now . But I made a slightly different changes as I'm not able to create sun moon toggle button as I suggested instead I added a different toggle .

**Comment by Aniket1026** [comment]
@markmead what should I be doing now ? I mean are these changes fine or should I be doing anything else ?

**Comment by markmead** [comment]
You need to re-request a review as I mentioned 😅
Screenshot 2022-08-02 at 14 46 07

**Comment by Aniket1026** [comment]
Sorry for the trouble @markmead I was unaware about how to re request review. Thanks for telling and I guess I've requested a review

**Comment by markmead** [comment]
No worries @Aniket1026 I was a little unsure myself a few months back!

File: data/components/application-ui-button-groups.mdx [hunk 5]
```diff
@@ -15,6 +15,7 @@ components:
     title: Action Group
   4:
     title: Action Group with Dropdown
+
```
Comment (markmead) [inline data/components/application-ui-button-groups.mdx:18]: Remove extra line.

File: data/components/application-ui-toggles.mdx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+category: application-ui
+seo:
+  title: Toggle Components
+  description: Application UI Toggle components created with Tailwind CSS
+components:
+  1:
+    title: Rounded Toggle
+  2:
+    title: Square Toggle  
+
```
Comment (markmead) [inline data/components/application-ui-toggles.mdx:14]: Remove extra line.

File: data/components/application-ui-toggles.mdx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+title: Toggles Group
+emoji: ⇌
```
Comment (markmead) [inline data/components/application-ui-toggles.mdx:3]: Should be using an emoji here, could be something like:

- 🎛
- 🎚
- 🕹

It doesn't have to be super accurate to what the collection is.

**Review by markmead — CHANGES_REQUESTED** [review]
Formatting is still off on the component HTML files.

There are some Tailwind CSS changes I'd like to make, if you look at https://play.tailwindcss.com/452s9Bp3Zs this should guide you.

File: data/components/application-ui-button-groups.mdx (hunk 5 above)
Comment (Aniket1026) [reply data/components/application-ui-button-groups.mdx]: Done

**Comment by Aniket1026** [comment]
The formatting is done and the new emoji is added . Extra lines are also removed but as for the changes you mentioned for some reason they're not reflecting in on the local host when I tried . So I simply changed the toggle color to green of the rounded toggle .

https://user-images.githubusercontent.com/92184189/182526263-2c465ac2-632c-4721-b73e-36af08522ea9.mp4

when I made the changes they don't seem to be working so the only change I made is change the toggle color of the rounded toggle.

**Comment by markmead** [comment]
Did you run `yarn watch` while developing or `yarn build` before pushing?

**Comment by Aniket1026** [comment]
ok It worked I was not doing running the yarn watch 😅😅

File: public/components/application-ui-toggles/1.html [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<div class="flex gap-8 p-8">
```
Comment (markmead) [inline public/components/application-ui-toggles/1.html:1]: Sorry, that's my bad, we don't need this line `flex gap-8 p-8`.

File: public/components/application-ui-toggles/2.html [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<div class="flex gap-8 p-8">
```
Comment (markmead) [inline public/components/application-ui-toggles/2.html:1]: Once again, we don't need this.

**Review by markmead — CHANGES_REQUESTED** [review]
Just two very small things to clean up and then we can get this merged! 🥳

**Comment by markmead** [comment]
Don't worry about making those changes @Aniket1026, it was my fault so I pulled it down and updated it.

**Comment by Aniket1026** [comment]
Hurray 🎉🎉. Really , thanks @markmead . You're a great mentor and I really enjoyed working on this . I learnedso many things in span of 3-4 days with your help 😊. Great experience 😄

**Pull request merged**